### PR TITLE
Update moloco.md

### PR DESCRIPTION
**Pull request by heonjang**
This updates the MCM's doc on the `Moloco MCM` integration.

### CODE REVIEW

**Comment by markzegarelli** [comment]
Thanks @heonjang!

I noticed that the Mapping chart table, the Amplitude Property column is empty. Is this expected?

File: content/collections/destination-catalog/en/moloco.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
 - You must enable this integration in each Amplitude project you want to use it in.
 - The Amplitude integration is only compatible with Moloco Commerce Media (MCM). 
 - You need a Moloco Commerce Media (MCM) account to enable this integration.
-- Amplitude matches the **User_id** to the ID field within Moloco Commerce Media (MCM) to associated events. If a user with that id doesn't exist within Moloco Commerce Media (MCM), a user is created. Make sure that the Amplitude **User_id** field matches the Moloco **Identity ID** to avoid user duplication.
+- Amplitude matches the **User_id** to the ID field within Moloco Commerce Media (MCM) to associated events. If a user with that id doesn't exist within Moloco Commerce Media (MCM), a user is created. Make sure that the Amplitude `User_id` field matches the Moloco **Identity ID** to avoid user duplication.
```
[the remark this answers is not in the document]
Comment (SpencerFleury) [reply content/collections/destination-catalog/en/moloco.md:32]: We use "ID" when abbreviating identification, unless it's part of a property name like `user_id`. There is an English language word "id," and while it's unlikely people will think we're talking about a concept from Freudian psychology in an article like this, we like to keep things as clear as possible all the same.

File: content/collections/destination-catalog/en/moloco.md [hunk 2]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
 
 To configure an Event Streaming integration from Amplitude to Moloco Commerce Media (MCM), you need the following information from Moloco Commerce Media (MCM):
 
-- **REST API Key:** To start sending data into Moloco Commerce Media (MCM), you first have to get your API Key. It is used to authenticate your requests to the API and connect the data with your account. Find this in your Moloco Commerce Media (MCM) account. 
+- **REST API Key:** To start sending data into Moloco Commerce Media (MCM), you need an API Key. Moloco uses this key to authenticate your requests to the API and connect the data with your account. Find this in your Moloco Commerce Media (MCM) account. 
 
 ### Amplitude setup
 
 1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:45]:
```suggestion
1. In Amplitude Data, click *Catalog* and select the *Destinations* tab.
```

Comment (SpencerFleury) [reply content/collections/destination-catalog/en/moloco.md:45]: Bold is for emphasis. UI elements are designated with italics.

File: content/collections/destination-catalog/en/moloco.md [hunk 3]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
 2. In the Event Streaming section, click **Moloco**.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:46]:
```suggestion
2. In the Event Streaming section, click *Moloco*.
```

File: content/collections/destination-catalog/en/moloco.md [hunk 4]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
+3. Enter a sync name using one of the following values, then click **Create Sync**. Create a sync for each event type
+  - `ADD_TO_CART`
+  - `ADD_TO_WISHLIST`
+  - `HOME`
+  - `ITEM_SEARCH_VIEW`
+  - `LAND`
+  - `PAGE_VIEW`
+  - `PURHCASE`
+  - `SEARCH`
 4. Toggle Status from **Disabled** to **Enabled**.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:56]:
```suggestion
4. Toggle Status from *Disabled* to *Enabled*.
```

File: content/collections/destination-catalog/en/moloco.md [hunk 5]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
-5. Paste your **REST API Key** and **Platform ID**.
+5. Fill in the following fields
+    1. **REST API Key**: This is the API key provided by moloco
+    2. **Platform ID**: Identifier of the platform (for example, `Moloco Demo Test`)
+    3. **Event Type** Type of the event this destination delivers. See the available options in step 3 above.
+    4. **Platform Name**: Name of the platform. Should be all lower-cased and replace underscore `_` and empty spaces with a hyphen `-`
 6. Toggle the **Send events**.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:62]:
```suggestion
6. Toggle *Send events*.
```

File: content/collections/destination-catalog/en/moloco.md [hunk 6]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
 7. In **Select and filter events** choose which events you want to send. Choose only the events you need in Moloco. *Transformed events aren't supported.*
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:63]:
```suggestion
7. In *Select and filter events* choose which events you want to send. Choose only the events you need in Moloco. **Transformed events aren't supported.**
```

Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:63]:
```suggestion
7. In *Select and filter events*, choose which events you want to send. Choose only the events you need in Moloco. **Transformed events aren't supported.**
```

File: content/collections/destination-catalog/en/moloco.md [hunk 7]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
 8. Under **Map properties to destination**, choose your user identifier and map specific Amplitude properties from Amplitude to Moloco.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:64]:
```suggestion
8. Under *Map properties to destination*, choose your user identifier and map specific Amplitude properties from Amplitude to Moloco.
```

File: content/collections/destination-catalog/en/moloco.md [hunk 8]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
 9. When satisfied with your configuration, click **Save**.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:65]:
```suggestion
9. When satisfied with your configuration, click *Save*.
```

File: content/collections/destination-catalog/en/moloco.md [hunk 9]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
+| `search_query`             | string | N        |             |                    | Query string for the search                                                                                                                                                                                   | microwave ovens                                                                   |
+| `items`                    | obj    | N        |             |                    | Items information. Details can be found below.                                                                                                                                                                |                                                                                   |
+
+## Use cases
+
+* **Real-time User Segmentation:** By sending streaming data from Amplitude, which is a comprehensive analytics platform, to Moloco, you can create real-time user segments based on various attributes and behaviors. This enables you to target specific user groups with personalized advertisements and optimize campaign performance accordingly.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:110]:
```suggestion
* **Real-time user segmentation:** By sending streaming data from Amplitude to Moloco, you can create real-time user segments based on various attributes and behaviors. This enables you to target specific user groups with personalized advertisements and optimize campaign performance accordingly.
```

File: content/collections/destination-catalog/en/moloco.md [hunk 10]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
+* **Dynamic Ad Creative Optimization:** By integrating Amplitude's streaming data with Moloco's machine learning engine, you can leverage real-time user insights to dynamically optimize ad creative elements. This includes dynamically adjusting ad content, images, offers, and calls to action based on user preferences and behaviors, resulting in more engaging and effective advertisements.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:111]:
```suggestion
* **Dynamic ad creative optimization:** By integrating Amplitude's streaming data with Moloco's machine learning engine, you can use real-time user insights to optimize ad creative elements. This includes dynamically adjusting ad content, images, offers, and calls to action based on user preferences and behaviors, resulting in more engaging and effective advertisements.
```

Comment (SpencerFleury) [reply content/collections/destination-catalog/en/moloco.md:111]: We never use "leverage" when "use" will suffice (which is almost always). Also, there were too many occurrences of "dynamically."

File: content/collections/destination-catalog/en/moloco.md [hunk 11]
```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
...
+* **Behavioral Retargeting:** Amplitude captures user behavior data across various touchpoints, providing valuable insights into user engagement and conversion patterns. By streaming this data to Moloco, you can implement behavioral retargeting strategies, where users who have shown specific behaviors or indicated interest in certain products or services are retargeted with relevant ads across different channels and devices.
```
Comment (SpencerFleury) [inline content/collections/destination-catalog/en/moloco.md:112]:
```suggestion
* **Behavioral retargeting:** Amplitude captures user behavior data across various touchpoints, providing valuable insights into user engagement and conversion patterns. By streaming this data to Moloco, you can implement behavioral retargeting strategies, where users who have shown specific behaviors or indicated interest in certain products or services are retargeted with relevant ads across different channels and devices.
```

**Comment by heonjang** [comment]
@SpencerFleury Thanks for the revision. I've applied your comments.

@markzegarelli Thanks for the catch. I've fixed the chart.

**Review by SpencerFleury — APPROVED** [review]

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**